packages/embeds/embed-react/playwright/tests/basic.e2e.ts

```diff
@@ -12,6 +12,7 @@ test.describe("React Embed", () => {
     }) => {
       const calNamespace = "inline";
       await embeds.gotoPlayground({ url: "/inline.html", calNamespace });
+      await page.waitForLoadState("networkidle");
```

Comment:
Since we know we are avoiding `networkidle`, and Playwright discourages its use, please remove it and find an alternative